server.js

```diff
@@ -123,12 +123,12 @@ function render (req, res) {
 
     const context = {
         SEOMixinTitle: {
-            title:'WebDollar - Currency of the Internet',
```

Comment:
Our motto is Currency of the internet, not of the people or something else, please rewview the code

Reply:
I did not change the motto.
But okay, I've got it done.